pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -7,8 +7,7 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 - forbidden! unless dashboard_user[:user_type] == 'teacher'
 - error_string_none_selected = I18n.t('dashboard_error_none_selected').gsub(/"/,"\\\\\"")
 - error_string_other_section = I18n.t('dashboard_error_other_section').gsub(/"/,"\\\\\"")
-- show_section_projects = DCDO.get('section_projects', CDO.section_projects)
-
+- show_section_projects = DCDO.get('section_projects', CDO.section_projects) || !!request.params['sectionProjects']
```

Comment:
`sectionProjects` query param is only used by UI tests, right? wonder if it's worth a comment here mentioning that explicitly.